app/cron.js

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable security/detect-non-literal-regexp */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-useless-constructor */
```

Comment:
That's a complicated question... Ideally, we should either remove or fix the rules for the whole project as a whole.